src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.html

```diff
@@ -19,7 +19,8 @@
                              (duplicateCurrentQuestionEvent)="duplicateCurrentQuestionHandler(i)"
                              (deleteCurrentQuestionEvent)="deleteExistingQuestionHandler(i)"
                              [(formModel)]="questionEditFormModels[i]"></tm-question-edit-form>
-    </div>
+  </div>
```

Comment:
Would you be able to fix the indentation here? This div should be the same level as the a tag and further indented than the div in line 24!